exercises/concept/rpn-calculator/.docs/instructions.md

```diff
@@ -0,0 +1,42 @@
+# Instructions
+
+Reverse Polish notation (RPN) is a way of writing mathematical expressions in which operators follow their operands.
+For example, instead of writing:
+```
+2 + 2
+```
```

Comment:
~~~suggestion

```
2 + 2
```

~~~

Markdown formatting: it's easiest when code blocks are surrounded by blank lines. Applies to the other blocks in this document also.

Reply:
Fixed in 7f6a9a4